staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -257,8 +257,9 @@ type VirtualMachineInstanceStatus struct {
 	// +optional
 	KernelBootStatus *KernelBootStatus `json:"kernelBootStatus,omitempty"`
 
-	// FSFreezeStatus is the state of the fs of the guest
-	// it can be either frozen or thawed
+	// FSFreezeStatus reflects whether a freeze or thaw command was executed,
+	// not the actual filesystem state. It’s unreliable for recovery,
```

Comment:
What does it mean: "It's unreliable for recovery" ?

Reply:
I'll improve it.